charts/kube-prometheus-stack/Chart.yaml

```diff
@@ -18,7 +18,7 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 16.14.1
+version: 16.14.2
```

Comment:
node_exporter major bump is mainly about helm2 compatibility which this chart is unaffected by, thus not raising minor here.